kolibri/core/hooks.py

```diff
@@ -37,24 +37,3 @@ def get_menu(self):
     class Meta:
 
         abstract = True
-
-
-class UserNavigationHook(KolibriHook):
```

Comment:
Removing the hook, might create others later on to bring in some of the other data that's going to populate the navigtion. As of right now, we don't really even know what it's going to look like.

This is one of the things I was skeptical about doing. I'd appreciate some input.